python/ray/util/client/server/proxier.py

```diff
@@ -150,18 +151,22 @@ def _get_redis_address(self) -> str:
         self._session_dir = connection_tuple["session_dir"]
         return self.redis_address
 
-    def _get_session_dir(self) -> str:
+    def _get_node(self) -> Any:
         """
         Gets the session_dir of this running Ray session. This usually
         looks like /tmp/ray/session_<timestamp>.
```

Comment:
is this correct?

Reply:
Yes, this is where logs always are.
The current one (or most recent one) is linked to `/tmp/ray/session_latest`

Reply:
Probably the doc string should be updated since we've changed the return type.

Reply:
Ahhh yes :)